packages/css/README.md

```diff
@@ -49,3 +49,15 @@ To use the compact mode, import the compact css **after** theme css, like so:
 import "@amsterdam/design-system-tokens/dist/index.css";
 import "@amsterdam/design-system-tokens/dist/compact.css";
 ```
+
+## Document styles
+
+Some elements of your document are out of reach for the design system components.
+However, their default styles or common boilerplate approaches can influence on the appearance on the CSS components.
+Apply the following to the document elements:
+
+The font-size must not be changed – e.g. through `html { font-size: 62.5% }` – because the typography system expects a rem to be 16 pixels.
```

Comment:
Can we make this part of `ams-body`? `html { font-size: 100% !important; }`

Reply:
Yeah, applying it to `:root` makes more sense. 

> We could, but isn’t asking people to add a class that overrides an explicit declaration more difficult than asking them to remove that declaration in the first place?

Hm, not sure. Asking them to remove something requires them to read and understand text and figure out if it's relevant for them. Adding an override to CSS you need to import anyway to get it to work doesn't.

Reply:
On second thought, maybe it's not such a good idea to enforce `font-size: 100%` with a css rule. If we add it to `:root`, `body` or `html`, projects that currently use `font-size: 62.5%` would have to refactor their entire project before being able to use our packages. 

If we just describe it, users can incrementally adopt our packages, having `font-size: 100%` on the parts that use our components, and `font-size: 62.5%` on the legacy parts. So maybe leave it as is?

Reply:
Very good point. Updated.